OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -2321,5 +2321,13 @@ public static DreamValue NativeProc_viewers(DreamObject instance, DreamObject us
             connection.WinSet(winsetControlId, winsetParams);
             return DreamValue.Null;
         }
+
+        [DreamProc("trimtext")]
+        [DreamProcParameter("text", Type = DreamValueType.String)]
+        public static DreamValue NativeProc_trimtext(DreamObject instance, DreamObject usr,
+            DreamProcArguments arguments)
+        {
+            return new DreamValue(arguments.GetArgument(0, "text").GetValueAsString().Trim());
```

Comment:
```suggestion
            return new DreamValue(arguments.GetArgument(0, "text").MustGetValueAsString().Trim());
```
`Get` is obsolete, all calls need to use `MustGet` or `TryGet`

Reply:
`TryGetValueAsString()` would be preferable here so you return `DreamValue.Null`, which is what BYOND does.